libs/dao-data/src/Profile.ts

```diff
@@ -23,13 +23,12 @@ type AccountProfile = {
   background?: string;
 };
 
-const CERAMIC_NODE = process.env['CERAMIC_NODE'] || '';
-
 export default class Profile {
   providers: Keychain;
 
-  constructor(providers: Keychain) {
+  constructor(providers: Keychain, node?: string) {
```

Comment:
Do we also need to update the Haus.ts create function to pass this in?

Reply:
I can handle this next week actually. I want to play with it.

Reply:
Yea, good point